src/pages/CreationPage.js

```diff
@@ -0,0 +1,21 @@
+import { CreationForm } from '../components/CreationForm'
+import { MoonCard } from '../components/MoonCard'
+import { MoonMsgBar } from '../components/MoonMsgBar'
+import { Rulebook } from '../components/Rulebook'
+
+export const CreationPage = () => {
+	return (
+		<main className='creation-page'>
+			<header className='creation-header'>
```

Comment:
personally I'd pull the header out of main. Apparently it can have aria implications.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#usage_notes

Reply:
I've always questioned if this was right or not. Thanks for the resource since I will now need to update a bunch of other projects. Would you wrap it in fragments then?

Reply:
yes to fragments

Reply:
Attempted this but I learned I am going to have to refactor App to make it so Header/Footer areas are _always_ fixed to the top & bottom body. Added to my to-do list